pkgs/development/python-modules/seabreeze/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, fetchurl
+, fetchFromGitHub
+, python3Packages
+}:
+
+## Usage
+# In NixOS, simply add the `udev` multiple output to services.udev.packages:
+#   services.udev.packages = [ pkgs.python3Packages.seabreeze.udev ];
+
+python3Packages.buildPythonPackage rec {
+  pname = "seabreeze";
+  version = "v0.6.0";
```

Comment:
It's not actually `v0.6.0`. You're fetching the latest commit.
Please adapt this according to the [manual](https://nixos.org/nixpkgs/manual/#sec-package-naming).